.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -3,6 +3,27 @@ name: Feature request
 about: Suggest an idea for this project
 
 ---
+**What part of Carbon does this feature request correlate to?**
+
+General graph logic:
+
+* [ ]  All graphs (axes, legends etc)
```

Comment:
should the content in the parentheses be moved to the next item `Graph Atrributes`?

Reply:
Now that I think about it, `All graphs` was originally meant to be the option for `Graph Attributes`. So I guess we can remove this `all graphs` or list some of the main attributes e.g:

* [ ]  Axes and ticks
* [ ]  Canvas
* [ ]  Labels
* [ ]  Legends
* [ ]  Popups
* [ ]  Regions
* [ ]  Other (please specify):

Reply:
I think trying to present a comprehensive list and maintain it will not be worth the effort.  The rest of the information on the PR should probably give us what we need so I think simply having `Graph attributes` is fine with me